### ISSUE TITLE
Are you sure that the batchnorm distorts features in layers? 

### ISSUE THREAD

**Issue by Ikhwansong** [proteus1991/GridDehazeNet#20]
Recently, i have a question that dose the batchnorm truly skews feature information?

In your paper, you commented "Following [19, 15], we do not use batch normalization" .  In the paper [19] and [15], however there is not the relevant comment or i  failed to find it.

In the [19], the author commented like that "We remove the batch normalization layers from our network as Nah et al.[19] resented in their image deblurring work". (Note that, the Nah et al [19] is corresponding to [15] you referred in your paper)

So finally, i looked into the paper [15], i.e., Deep Multi-scale Convolutional Neural Network for Dynamic Scene Deblurring.
In this paper, the author said "We did not use batch normalization layers since we trained model with mini-batch of size
2, which is smaller than usual for batch normalization.". 

They suggested that because the batch size is too small than the general used size, the model didn't use batch normalization.

I think. your and their view for why the batchnorm is improper for a model, were slightly different. Your default batch size is the number of 24. It is too big so that the batchnorm was necessary. 

Why didn't use it? and did you perform the experiment with batchnorm?

**Comment by proteus1991** [comment]
Hi, sorry for the late reply. I'm quite busy recently for my graduation. 

To my own understanding (may not be very accurate or theoritically proved), the batch normalization is usually not used in **low-level vision tasks** including, among others, dehazing, deblurring and super-resolution since the batch normalization in some extent makes the image _blurry_ by _averaging_ the batch. Therefore, it may deteriorate the restored result.

If you have a diffferent perspective, please feel free to contact me.
